tests/conftest.py

```diff
@@ -63,6 +70,27 @@ def log_has_re(line, logs):
                   False)
 
 
+def num_log_contains(line, logs, count=False):
+    # caplog mocker returns log as a tuple: ('freqtrade.something', logging.WARNING, 'spamfoobar')
+    # and we want to check how many times line ('foo', for example) is contained in the tuples.
+    return sum(line in logger_name or line in message
```

Comment:
You'll definitely not want to match "line" against both logger_name and message.

if you think there is a need to know where the log message comes from (logger_name) - this should be a separate argument.
I've yet to encounter a case where i care about that though ...

Reply:
Removed the logger_name check.